easybuild/easyconfigs/m/Meep/Meep-1.6.0-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'Meep'
+version = '1.6.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://meep.readthedocs.io/en/latest/'
+description = """Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package
+ developed at MIT to model electromagnetic systems."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True, 'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['https://github.com/stevengj/meep/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c1808d5affe75b40fe7c61f9807377ce690ede20929acbcca09614345e55fb26']
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('M4', '1.4.18'),
+    ('pkg-config', '0.29.2'),
+    ('SWIG', '3.0.12', versionsuffix)
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('MPB', '1.6.2', versionsuffix),
+    ('Harminv', '1.4.1'),
+    ('HDF5', '1.10.1'),
+    ('libctl', '4.1.3'),
+    ('libGDSII', '0.1'),
+    ('GSL', '2.4'),
+    ('Guile', '1.8.8'),
+]
+
+#  taken from autogen.sh in root source directory
+preconfigopts = 'autoreconf --verbose --install --symlink --force &&'
+preconfigopts += 'autoreconf --verbose --install --symlink --force &&'
+preconfigopts += 'autoreconf --verbose --install --symlink --force &&'
```

Comment:
@hajgato Again?!

Reply:
aken from the autogen.sh. did not check whether is it necessary.